software_reseller/__manifest__.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+{
+    'name': 'Software Reseller',
+    'version': 'saas~16.4.1.0',
+    'category': 'Studio',
+    'description': u"""
+This module has been generated by Odoo Studio.
+It contains the apps created with Studio and the customizations of existing apps.
+""",
+    'author': 'Odoo',
+    'depends': [
+        'account_followup',
```

Comment:
It should be possible to clean this depends

Reply:
Done